managed/ui/src/components/universes/UniverseDetail/UniverseDetail.js

```diff
@@ -422,6 +426,28 @@ class UniverseDetail extends Component {
       </div>
     );
 
+    const toggleUniverseBody = (
+      <>
```

Comment:
why don't you put this content into the `ToggleUniverseStateContainer` component?
this seems to be an essential part of that modal dialog, not sure why it's not there

Reply:
I actually took an example of the 'Delete Universe' container component. In that, we're passing 1 prop 'body' which supposed to have the modal body content. So, I just follow that one,, and I'm passing the modal body content for the 'Toggle Universe' container from the parent itself.